SocketIOClientSwift/SocketInitialOptions.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  InitialOptions.swift
+//  Socket.IO-Client-Swift
+//
+//  Created by Lukas Schmidt on 17.10.15.
+//
+//
+
+import Foundation
+
+struct SocketInitialOptions {
```

Comment:
If you make this an `enum` with associated `String` values instead, you can statically guarantee that all keys are unique.